PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/model/GeoCoordinate.java

```diff
@@ -0,0 +1,87 @@
+package edu.cascadia.mobas.photopoints.model;
+
+import androidx.annotation.NonNull;
+import com.google.android.gms.maps.model.LatLng;
+
+
+
+public class GeoCoordinate {
```

Comment:
Java has a built-in class, Point, that looks a lot like a simple form of this. Could have potentially used that as a parent. No show-stopper though.

Reply:
I attempted to do just that, but it is a final class, can not be used as a parent.